### DISCUSSION TITLE
How to trace back to admin portals through mixcore CMS?

### DISCUSSION THREAD

**Discussion by docthoi** [mixcore/mix.core#652] [Q&A] [answered]
Hi, recently I've received a legacy website containing all the CMS portals for our websites, made in mixcore. I was tasked with finding out how to regain control from an ex-employee who built these portals using mixcore. We're worried he still has control of our websites after he had left the company. 

Is there a way to gain access to the admin portal from the CMS website? Right now, we're only left with id and passwords for individual websites connected to the mixcore engine. Thanks in advance.

**Comment by Smilefounder** [comment, marked as answer]
Hi @docthoi, are you able to send us any information via contact@mixcore.org? We will take a look as we will not able to provide the way to "hack" admin account if we are not so sure.

**Reply by docthoi** [reply]
Of course, I'll let my boss first, please wait a few days.
